core/sql/executor/ExHbaseIUD.cpp

```diff
@@ -4193,7 +4188,7 @@ ExWorkProcRetcode ExHbaseAccessSQRowsetTcb::work()
 	      step_ = DONE;
 	  }
 	  break;
-      case ASYNC_INSERT_COMPLETE:
+      case ASYNC_OPERATION_COMPLETE:
```

Comment:
Would it be better to call this step "COMPLETE_ASYNC_OPERATION"? ASYNC_OPERATION_COMPLETE sounds as if the async operation is already complete, but what it really means is that we need to complete the operation.

Don't want to hold up this change, if you agree then maybe you can rename it in some future checkin.